src/app/routes/enrolment/view-requests/view-requests.component.html

```diff
@@ -20,6 +20,24 @@ <h3 class="default-color mb-2">{{ claim?.roleName | uppercase }}</h3>
       </app-fields>
 
       <app-fields data-qa-id="issuer-fields" title="Issuer fields" [fieldsList]="issuerFields$ | async"> </app-fields>
+      <ng-container *ngIf="claim?.isIssued && claim?.verifiableCredential">
```

Comment:
Personally i prefer to have all logic in typescript files, so you could create a getter:
```
get isRawVerifiableVisible(): boolean {
  return this.claim?.isIssued && this.claim?.verifiableCredential
}
```
However it comes from times when i did not test html in units. You already tested this logic so it's up to you if you want to move it into getters.

Reply:
@dawidgil I removed this and moved logic into enrolmentClaim (and added several tests)